api/clients/v2/examples/client_construction.go

```diff
@@ -31,6 +31,8 @@ const (
 	ethRPCURL                        = "https://ethereum-holesky-rpc.publicnode.com"
 	disperserHostname                = "disperser-testnet-holesky.eigenda.xyz"
 	certVerifierAddress              = "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162"
+	certVerifierRouterAddress        = "0x7F40A8e1B62aa1c8Afed23f6E8bAe0D340A4BC4e"
```

Comment:
do we still need the certVerifierAddress when the router is input?

Reply:
[92ab00c](https://github.com/Layr-Labs/eigenda/pull/1532/commits/92ab00cee909375647e0c4177761bd799429dfda)